eng/scripts/typespec/Check-CodeGeneration.ps1

```diff
@@ -12,10 +12,28 @@ try {
     Write-Host 'Checking generated files difference...'
     git -c core.safecrlf=false diff --ignore-space-at-eol --exit-code
     if ($LastExitCode -ne 0) {
-        Write-Error 'Generated codes are not up to date. Please run: eng/packages/http-client-csharp/eng/scripts/Generate.ps1'
+        Write-Error 'Generated codes of Azure plugin are not up to date. Please run: eng/packages/http-client-csharp/eng/scripts/Generate.ps1'
```

Comment:
We should avoid using "plugin". Instead use "generator" or "emitter" depending on the context.

Reply:
rephrased